noir-projects/aztec-nr/aztec/src/keys/point_to_symmetric_key.nr

```diff
@@ -13,6 +13,7 @@ pub fn point_to_symmetric_key(secret: Scalar, point: Point) -> [u8; 32] {
     shared_secret_bytes_with_separator =
         arr_copy_slice(shared_secret, shared_secret_bytes_with_separator, 0);
     shared_secret_bytes_with_separator[32] = GENERATOR_INDEX__SYMMETRIC_KEY;
+    // TODO(https://github.com/AztecProtocol/aztec-packages/issues/10537): Replace use of sha256
```

Comment:
sneaked this in as discussed [here](https://aztecprotocol.slack.com/archives/C06FTSX4NV7/p1733753694682129?thread_ts=1733231448.996449&cid=C06FTSX4NV7)